pkgs/by-name/an/angular-language-server/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  stdenv,
+  fetchYarnDeps,
+  fetchFromGitHub,
+  nodejs,
+  yarnConfigHook,
+  yarnInstallHook,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "angular-language-server";
+  version = "18.2.0";
+
+  src = fetchFromGitHub {
+    owner = "angular";
+    repo = "vscode-ng-language-service";
+    rev = "v${version}";
+    hash = "sha256-9+WWKvy/Vu4k0BzJwPEme+9+eDPI1QP0+Ds1CbErCN8=";
+  };
+
+  offlineCache = fetchYarnDeps {
+    yarnLock = src + "/yarn.lock";
+    hash = "sha256-N0N0XbNQRN7SHkilzo/xNlmn9U/T/WL5x8ttTqUmXl0=";
+  };
+
+  nativeBuildInputs = [
+    yarnConfigHook
+    yarnInstallHook
+  ];
+
+  buildInputs = [
+    nodejs
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+
+    tsc -b && yarn bazel build :npm
+
+    runHook postBuild
+  '';
+
+  meta = {
+    description = "lsp for angular completions, AOT diagnostic, quick info and go to definitions";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

Reply:
Was this just for capitalization? Otherwise it's basically taken from the README.

Reply:
Yes